recipes/freecad/build.sh

```diff
@@ -0,0 +1,47 @@
+mkdir -p build
+cd build
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [ `uname` = "Darwin" ]; then
+
+      sed -i '' 's/Xcode-9.app/Xcode.app/g' $PREFIX/lib/cmake/opencascade/OpenCASCADEVisualizationTargets.cmake
```

Comment:
This patch probably should land in the `opencascade` recipe.

Reply:
I guess this can be removed once we build osx on circle-ci.